app/stac_api/validators.py

```diff
@@ -1,14 +1,21 @@
+import json
 import logging
 import re
 from datetime import datetime
 
 import multihash
 
+from django.contrib.gis.gdal.error import GDALException
 from django.contrib.gis.geos import GEOSGeometry
+from django.contrib.gis.geos import Point
+from django.contrib.gis.geos import Polygon
 from django.core.exceptions import ValidationError
 from django.utils.translation import gettext_lazy as _
 
+from rest_framework.exceptions import ValidationError as RestValidationError
```

Comment:
See my comment below but this is not needed anymore if moved into validators_serializer.py